### ISSUE TITLE
Documentation is missing

### ISSUE THREAD

**Issue by mycroes** [ButchersBoy/Dragablz#79]
Though there exists some documentation on the blog and some documentation in the sample projects, it's hard to find how to use certain features. It would be nice to have some documentation on (at least) the InterTabClient (and IInterTabClient), Layout and NewItemFactory.

**Comment by ButchersBoy** [comment]
Yeah, I am slowly trying to work to improve code comments, then will hopefully generate a website out of that.  Most of my work goes into getting the thing working then providing examples, but the API is reasonably settled now so I will try and improve.

**Comment by mycroes** [comment]
I'd rather see some wiki pages than code comments; the code is quite clear, the naming is OK or reasonably good (didn't dive into the internal naming, but the external API is quite concise naming-wise). What I'm missing is why I need my own IInterTabClient (spoiler: because I use Caliburn Micro and I need to bind my ViewModel) and a page that says that NewItemFactory is a Func<object> (or Func<MyTabItem>) which maybe also mentions that there's no way to cancel the new tab creation in that func (or is there?).

I might be able to contribute some documentation, like the Caliburn Micro InterTabClient that I have...

**Comment by ButchersBoy** [comment]
Hey yeah, if you have a specific example for Caliburn, please feel free to add it to the Wiki.  It would be appreciated.

**Comment by ButchersBoy** [comment]
> NewItemFactory is a Func (or Func) which maybe also mentions that there's no way to cancel the new tab creation in that func (or is there?)

Please feel free to raise a new issue/request for this, it sounds like a positive improvement to the API, so long as I don't break existing users.

**Comment by mycroes** [comment]
Just started on the documentation: https://github.com/ButchersBoy/Dragablz/wiki/Dragablz-Basics

**Comment by WillCohenInfotrack** [comment]
Hi MyCroes, Could you post your InterTab Client for caliburn micro, having the same issue here.

**Comment by ButchersBoy** [comment]
@mycroes :+1: Thankyou for contribution! :smile:

**Comment by mycroes** [comment]
@WillCohenInfotrack Added it to the wiki: https://github.com/ButchersBoy/Dragablz/wiki/InterTabClient .  I did analog copy of the code (laptop -> eyes -> hands -> keyboard), so if it doesn't work it's my fault ;-)

**Comment by ButchersBoy** [comment]
Looks good!  I don't use Caliburn so it's a big help.

**Comment by mycroes** [comment]
I was hoping to have already written more documentation already, but progress on my application stalled a bit since I'm having trouble finding time to work on it, thus also delaying any documentation work. I hope it's enough of a start to get some more info on the wiki.

**Comment by ButchersBoy** [comment]
ha, it's not so easy to keep on top of documentation :wink:

**Comment by mbowles201** [comment]
Many of us use the ShellView (Window) and its "ActiveItem" and let  Conductors to the work of swapping out the ViewModel (and View) that sit in the ConentControl.   Thus you can show a Splace screen, then your MainView, which is a UserControl and the primary Screen of the app.  This Screen is where the Drabablz tab control would sit and each tab would house a different ViewModel (and its View).   Some ideas and documentation on doing that would be greatly appreciated
